sandbox.go

```diff
@@ -427,7 +427,13 @@ func (sb *sandbox) ResolveIP(ip string) string {
 }
 
 func (sb *sandbox) ExecFunc(f func()) error {
-	return sb.osSbox.InvokeFunc(f)
+	sb.Lock()
+	osSbox := sb.osSbox
+	sb.Unlock()
+	if osSbox != nil {
+		return sb.osSbox.InvokeFunc(f)
```

Comment:
You are still accessing `sb.osSbox` outside lock.
That could have been set to nil before `sb.osSbox.InvokeFunc(f)` gets a chance to run